ignite/contrib/handlers/polyaxon_logger.py

```diff
@@ -160,10 +269,10 @@ class OptimizerParamsHandler(BaseOptimizerParamsHandler):
         tag (str, optional): common title for all produced plots. For example, "generator"
     """
 
-    def __init__(self, optimizer, param_name="lr", tag=None):
+    def __init__(self, optimizer: Optimizer, param_name: str = "lr", tag: str = None):
         super(OptimizerParamsHandler, self).__init__(optimizer, param_name, tag)
 
-    def __call__(self, engine, logger, event_name):
+    def __call__(self, engine: Engine, logger: PolyaxonLogger, event_name: Any):
```

Comment:
is it right here `event_name: Any` or should it be `event_name: str`?

Reply:
Making it `Union[CallableEventWithFilter, Enum]`.